packages/build/src/buildbot_client/main.js

```diff
@@ -0,0 +1,76 @@
+const net = require('net')
+
+const { addErrorInfo } = require('../error/info')
+
+const startBuildbotClient = function(buildbotServerSocket) {
+  return new Promise((resolve, reject) => {
+    try {
+      const client = net.createConnection(buildbotServerSocket)
+      client.on('connect', () => resolve(client))
+    } catch (error) {
+      addErrorInfo(error, {
+        type: 'buildbotClientStartup',
+        location: { buildbotServerSocket },
+      })
+      reject(error)
+    }
+  })
+}
+
+const closeBuildbotClient = function(client) {
```

Comment:
Should the `client` variable in this file be named `buildbotClient` to differentiate it from other clients?